editor/components/inserter/menu.js

```diff
@@ -71,6 +71,10 @@ export class InserterMenu extends Component {
 		this.switchTab = this.switchTab.bind( this );
 	}
 
+	componentDidMount() {
+		this.props.fetchReusableBlocks();
```

Comment:
Noting that this can become quite noisy for AJAX requests: Every time the user opens an inserter.

Personally I'm okay with this, since it ensures that the data is kept fresh, but worth keeping in mind.

Reply:
For now I think this is OK. If it's a problem then we can limit it to fetch once when the inserter is opened for the first time.